libs/ledger-live-common/src/exchange/swap/types.ts

```diff
@@ -27,41 +27,70 @@ export type ExchangeRate = {
   // There's a delta somewhere between from times rate and the api.
   rateId?: string;
   provider: string;
-  providerType: "CEX" | "DEX";
+  providerType: ExchangeProviderType;
   tradeMethod: "fixed" | "float";
   error?: Error;
   providerURL?: string;
   expirationTime?: number;
 };
 
-type ExchangeRateV5ProviderType = "DEX" | "CEX" | "DISABLED";
+type ExchangeProviderType = "CEX" | "DEX";
 
-type ExchangeRateV5CommonProperties = {
+type ExchangeRateV5CommonPropertiesRaw = {
```

Comment:
As we are not using "properties" in the other types we could rename it as `ExchangeRateV5CommonRaw` (suggestion, not required)

Reply:
Yes, good suggestion.